src/overview/helpers.ts

```diff
@@ -17,4 +18,4 @@ export function safeDecodeURI(data: string): string {
   } catch (error) {
     return data
   };
-};
\ No newline at end of file
+};
```

Comment:
i'm admittedly no typescript expert - but I think this type of helper function to essentially further-check a data structure that we've also defined a custom type/interface for is maybe a good fit for a user-defined type guard like described here? https://basilebong.com/blog/custom-type-guards-in-typescript-dg8

had to read this one over a couple times to understand it & took me a bit to wrap my head around why the `PlanXExportData[]` type isn't valid "enough" on its own - maybe the type guard would help with clarity? just a thought, might not be the right approach!